### ISSUE TITLE
Update prototype content for research

### ISSUE THREAD

**Issue by eileen-coforma** [department-of-veterans-affairs/va.gov-team#75876]
## Value Statement
**_As a_** designer/researcher
**_I want to_** fix known areas of friction within the prototype
**_So that_** I can make the most informed recommendations

---
## Test Notes and Design Links
- Confirmation page [[FIGMA](https://www.figma.com/file/BplQkEDZzD8NPPNmLrhvXv/Request-a-Board-Appeal-%2F-Notice-of-Disagreement-(VA-10182)?type=design&node-id=264-9630&mode=design&t=Xky3XcKBb7bGlpgz-11)]

## Acceptance Criteria
- [ ] New confirmation page displays content as shown in Figma file
- [ ] Date for claim submission is dynamically generated to be today's date
- [ ] Expiration date for PDF is dynamically generated to be 7 days after today's date
- [ ] Prototype is checked for bugs on Chrome AND Safari
- [ ] DK has done a11y checks for new design

## Tech Tasks
- [ ] _Task_
- [ ] _Task_
- [ ] _Task_

## How to Measure Success
[Assumptions document](https://docs.google.com/document/d/13iGifzoENfoNf2Vn8gimDX_udRVSKXaUYZNGdy4H1bs/edit) is updated with new criteria.

## Out of scope
- Add here

## Definition of Ready 

- [ ] Clear value description
- [ ] Test cases defined
- [ ] Approved designs and success measure(s) included
- [ ] Sample data provided where appropriate
- [ ] Fits within the sprint
- [ ] Dependencies linked

## Definition of Done

- [ ] Test cases passed
- [ ] E2E testing updated
- [ ] Unit testing updated (90% coverage)
- [ ] Integration testing updated (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Reviewed and approved by product and/or design

**Comment by Mottie** [comment]
PR is open, reviewed. & ready to be merged... blocked by failing CI errors unrelated to our app.

**Comment by eileen-coforma** [comment]
New single confirmation page prototype design is ready. https://www.figma.com/file/BplQkEDZzD8NPPNmLrhvXv/Request-a-Board-Appeal-%2F-Notice-of-Disagreement-(VA-10182)?type=design&node-id=264-9630&mode=design&t=okhgBivcZlv0iTGH-11

**Comment by eileen-coforma** [comment]
We'll need to refine the AC and description based on the pivot.

**Comment by eileen-coforma** [comment]
Copying old description into a comment as archive

## Value Statement
**_As a_** designer/researcher
**_I want to_** fix known areas of friction within the prototype
**_So that_** I can make the most informed recommendations

---
## Test Notes and Design Links
- Confirmation Page ([FIGMA](https://www.figma.com/file/BplQkEDZzD8NPPNmLrhvXv/Request-a-Board-Appeal-%2F-Notice-of-Disagreement-(VA-10182)?type=design&node-id=0-8647&mode=design&t=FGF6O9CVoINzbPby-11))
- Changes are to the link language, and an additional H4
- Submission review page ([FIGMA](https://www.figma.com/file/BplQkEDZzD8NPPNmLrhvXv/Request-a-Board-Appeal-%2F-Notice-of-Disagreement-(VA-10182)?type=design&node-id=0-8909&mode=design&t=FGF6O9CVoINzbPby-11))
- Mock data for uploaded evidence, the .pdf name

## Acceptance Criteria
- [ ] Confirmation page has new header and updated link language
- [ ] Submission review page has new mock data for file name
- [ ] Prototype tested on Safari and Chrome to ensure there aren't bugs

## Tech Tasks
- [ ] _Task_
- [ ] _Task_
- [ ] _Task_

## How to Measure Success
_- Define the expected result and where to measure it._

## Out of scope
- Add here

## Definition of Ready 

- [ ] Clear value description
- [ ] Test cases defined
- [ ] Approved designs and success measure(s) included
- [ ] Sample data provided where appropriate
- [ ] Fits within the sprint
- [ ] Dependencies linked

## Definition of Done

- [ ] Test cases passed
- [ ] E2E testing updated
- [ ] Unit testing updated (90% coverage)
- [ ] Integration testing updated (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Reviewed and approved by product and/or design

---
## How to configure this issue
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Status** (`Needs-Refinement`, `Needs-Estimate`, `discovery`, etc.)

**Comment by eileen-coforma** [comment]
@Mottie Email copy has also been approved. Good to go with the prototype as designed.

**Comment by saderagsdale** [comment]
@eileen-coforma @davidakennedy  Let me know when you've had a moment to look at the prototype for accessibility and content plz.

**Comment by eileen-coforma** [comment]
Content looks good to me ✅

**Comment by HeatherWidmont** [comment]
@davidakennedy took a look & says it looks good. Validated & ready to close
